lms/djangoapps/django_comment_client/utils.py

```diff
@@ -805,9 +805,6 @@ def is_discussion_enabled(course_id):
     """
     Return True if Discussion is enabled for a course; else False
     """
-    if settings.FEATURES.get('CUSTOM_COURSES_EDX', False):
-        if get_current_ccx(course_id):
```

Comment:
@mtyaka There's a leftover import for `get_current_ccx` at the top of the file that you'll need to remove.